api/trace/span.go

```diff
@@ -167,12 +166,24 @@ func (sp *span) Tracer() Tracer {
 	return sp.tracer
 }
 
-func (sp *span) Log(ctx context.Context, msg string, args ...core.KeyValue) {
-	log.With(sp).Log(ctx, msg, args...)
+func (sp *span) AddEvent(ctx context.Context, msg string, attributes ...core.KeyValue) {
```

Comment:
See how we solve this in Java. We have an interface called Event that can be implemented as "EventF" or as "EventString" so the main Span Interface does not have to provide all these variants.

Reply:
done.